src/scripts/GameController.ts

```diff
@@ -410,6 +410,41 @@ class GameController {
                         return e.preventDefault();
                     }
                     break;
+                case Settings.getSetting('hotkey.oakItems.loadout.1').value:
+                    // Activate to Oak Item Loadout 1
+                    if (oakItems.canAccess() && !$oakItemsModal.data('disable-toggle')) {
+                        App.game.oakItemLoadouts.activateLoadout(0);
+                        return e.preventDefault();
+                    }
```

Comment:
This can be solved cleaner. See how buttons inside towns works

Reply:
I assume you mean how we handle hotkey presses in towns on lines 343-372. I'm not sure I follow how that could be cleaner. If you are referring to the if condition, I was using the same condition used for opening the oak items modal. If you are referring to the handling of number key presses to select TownContent/NPCs, that would require tying the hotkeys to the number keys. With the current changes it is set up so you could set them to anything. Would you prefer if it was always set to 1-5 (with a modifier) and it wasn't configurable?